patches/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java.patch

```diff
@@ -1,5 +1,14 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/ActiveRenderInfo.java
+@@ -28,7 +28,7 @@
+     private static float field_74587_g;
+     private static float field_74596_h;
+ 
+-    public static void func_74583_a(EntityPlayer p_74583_0_, boolean p_74583_1_)
++    public static void updateRenderInfo(Entity p_74583_0_, boolean p_74583_1_)
```

Comment:
This still breaks binary compatibility as the method has a different name. Anyone calling the old method will get an exception.  If you need the new header, keep a copy of the old one and patch it to call the new one